api/src/routes/filters.rs

```diff
@@ -119,7 +161,7 @@ fn build_filter_bson(filter: Filter) -> Result<mongodb::bson::Document, mongodb:
     let database_filter_bson = doc! {
         "_id": id,
         "group_id": 41, // consistent with other test filters
-        "filter_id": filter.id,
+        "id": filter.id,
```

Comment:
Does the filter worker depend on this schema?

Reply:
Yeah though the group_id can be removed. If you edit the schema that's fine, we should just make the associated changes in boom. Can be done in separate PRs if that's easier.